### PR TITLE
updgrade and cleanup/lint fixes

### PR DESCRIPTION
**Pull request by saraswatpuneet**
## Details
- Upgrade depdencies minus polkadotjs and related 
- Remove as much unused packages
- Update lint and address errors

### CODE REVIEW

File: libs/common/src/interfaces/dsnp.ts
Comment (wilwade) [inline libs/common/src/interfaces/dsnp.ts:1]: I think we want to keep these as types. Types are more restrictive and I think we want that

File: libs/common/src/utils/queues.ts
Comment (wilwade) [inline libs/common/src/utils/queues.ts:1]: Surprised this didn't reformat the file.

File: libs/common/src/utils/redis.ts
Comment (wilwade) [inline libs/common/src/utils/redis.ts:1]: Surprised this didn't reformat the file.

**Review by wilwade — COMMENTED** [review]
Three questions, but otherwise looks good.

File: libs/common/src/utils/queues.ts
Comment (saraswatpuneet) [reply libs/common/src/utils/queues.ts:1]: Ran prettier

File: libs/common/src/interfaces/dsnp.ts
Comment (saraswatpuneet) [reply libs/common/src/interfaces/dsnp.ts:1]: Disabled consistent type checking in eslint to prevent auto fixing type to interface

**Review by wilwade — APPROVED** [review]

**Pull request merged**